### ISSUE TITLE
GUI crashing when loading icon

### ISSUE THREAD

**Issue by bitjungle** [Jakub-Konkol/Porchlight#1]
Tested under Python 3.11.0, MacOS 13.1. Traceback below.

```
$ python -m porchlight
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/opt/homebrew/Caskroom/miniforge/base/envs/sci/lib/python3.11/site-packages/porchlight/__main__.py", line 7, in <module>
    oop = OOP()
          ^^^^^
  File "/opt/homebrew/Caskroom/miniforge/base/envs/sci/lib/python3.11/site-packages/porchlight/porchlight.py", line 144, in __init__
    self.create_widgets()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/sci/lib/python3.11/site-packages/porchlight/porchlight.py", line 409, in create_widgets
    icon_base64 = base64.b64encode(icon_gif.read())
                                   ^^^^^^^^^^^^^^^
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf4 in position 6: invalid continuation byte
```

**Comment by bitjungle** [comment]
I have a suggestion for a fix, see pull request #2

**Comment by Jakub-Konkol** [comment]
Thanks for letting me know, and proposing a modification! I hadn't had a chance to test on Mac OS yet. I have approved your modification and merged it into the main branch. I'll request someone I know to try running it on their Mac device and see if it works for them as well.
